master/buildbot/newsfragments/sqlalchemy-1.1.0.bugfix

```diff
@@ -1 +0,0 @@
-SQLAlchemy 1.1.0 or higher is now required for Buildbot, in order to work with Python 3.5+.
```

Comment:
Should this one end up on Buildbot 2.0 relnotes?

Reply:
well it is not very important. I chose not to take it